build.gradle.kts

```
@@ -55,8 +55,9 @@ dependencies {
     // https://mvnrepository.com/artifact/org.languagetool/language-ru
     implementation("org.languagetool", "language-ru", "4.2")
 
-    // https://mvnrepository.com/artifact/commons-cli/commons-cli
-    implementation("commons-cli", "commons-cli", "1.4")
+    implementation("info.picocli", "picocli", "4.2.0")
+    annotationProcessor("info.picocli", "picocli", "4.2.0")
```

```suggestion
    annotationProcessor("info.picocli", "picocli-codegen", "4.2.0")
```

https://github.com/remkop/picocli/tree/master/picocli-codegen#213-gradle